### DISCUSSION TITLE
WMSLayer not registered in Pydeck

### DISCUSSION THREAD

**Discussion by Javier-Jimenez99** [visgl/deck.gl#8054] [Q&A]
I'm trying to use CMEMS WMS layers like this one in python: [https://nrt.cmems-du.eu/thredds/wms/cmems_mod_glo_phy-cur_anfc_0.083deg_P1D-m?request=GetCapabilities&service=WMS](url)

This is the code I used to define the layer:
```
wms_layer = pdk.Layer(
    type="WMSLayer",
    data=wms_url,
    layers=["thetao"],
    service_type="wms",
    pickable=False,
)
```

It doesn't appear into the map so I checked the console and it says: 
`JSON converter: No registered class of type _WMSLayer({"data":"https://nrt.cmems-du.eu/thredds...)  `
![imagen](https://github.com/visgl/deck.gl/assets/38747614/5858b23e-0556-4605-a90f-4ea08b1ab6ff)

I thought that all the layers added to deck.gl would be automatically added to pydeck too. Is there any workaround to fix this?

**Comment by ibgreen** [comment]
FWIW: it is an experimental layer and it is exported with underscore, which I would expect to be preserved during auto registration of layer exports https://github.com/visgl/deck.gl/blob/master/modules/geo-layers/src/index.ts#L22

Maybe try:

```python
wms_layer = pdk.Layer(
    type="_WMSLayer",
```

**Reply by Javier-Jimenez99** [reply]
Thanks for your response but I've tryed it and it still doesn't work

**Reply by elbys** [reply]
Hi,

I am experiencing the same problem...

Any ideas how can we access the experimental layers from pydeck ?
